xbmc/ServiceBroker.h

```diff
@@ -135,9 +136,14 @@ class CServiceBroker
   static void RegisterAppPort(std::shared_ptr<CAppInboundProtocol> port);
   static void UnregisterAppPort();
 
+  static std::shared_ptr<CDPMSSupport> GetDPMSManager();
```

Comment:
there won't be a DPMSManager without a winsystem. This can be added as a subcomponent to WinSystem

Reply:
^^